manifests/init.pp

```diff
@@ -81,6 +82,7 @@
   Puppetboard::Syslogpriority $python_loglevel                = 'info',
   Optional[String[1]] $python_proxy                           = undef,
   Optional[String[1]] $python_index                           = undef,
+  Boolean $python_systempkgs                                  = false,
```

Comment:
Maybe we should not change the default value a user might have set globally with:

```puppet
Python::Pyvenv {
  systempkgs => true,
}
```

…and do not set an explicit value unless one is passed:

```suggestion
  Optional[Boolean] $python_systempkgs                        = undef,
```

Reply:
Also wondering if the name should not rather contain `pyvenv` since it applies only to `python::venv` and we have other `python` things… What do you think @h0tw1r3 ?

Reply:
I don't actually use puppetboard anymore, just rebasing this to get it off my list :)

A default of `false` makes the most sense to me because it's the current behavior.
The variable naming follows the current convention.

I could make sense to review and change both of these, but that would be a breaking change.
This pull request has a very limited scope.

Reply:
w00t, my bad, I missed the previous value in the diff! You are right, this would change the default!